wordpress/wp-oem/Presslabs/ObjectCache/Memcached.php

```diff
@@ -146,6 +174,47 @@ public static function mark_updated_option($option)
         }
     }
 
+    private function preloadCache()
+    {
+        if ((defined('WP_CLI') && WP_CLI) || (defined('DOING_CRON') && DOING_CRON)) {
+            return;
+        }
+
+        $this->request_hash = md5(json_encode(array(
```

Comment:
Don't we also need the HTTP method?

I'm thinking about admin-ajax requests: for `get` there are other keys than for `post`.
In this case, the payload is also important since some admin-ajax actions will need totally different cache keys preloaded. Yet, maybe for the majority of the time, is good enough.

Reply:
Probably request method makes sense. Actually I would do it just for GET's at the moment.